ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceVisibilityIT.scala

```diff
@@ -28,9 +28,21 @@ import com.daml.test.evidence.tag.Security.SecurityTest.Property.Privacy
 
 import scala.collection.immutable.Seq
 import scala.collection.mutable
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration.DurationInt
 
 class TransactionServiceVisibilityIT extends LedgerTestSuite {
 
+  // triple eventually wait duration compared to default to avoid database timeouts
+  // when running against Oracle in enterprise mode
+  def eventually[A](
+      assertionName: String
+  )(runAssertion: => Future[A])(implicit ec: ExecutionContext): Future[A] =
+    Eventually.eventually(
+      assertionName,
+      firstWaitTime = 30.millis, // compared to default of 10.millis
```

Comment:
are you sure that this is the problem? this seems to me already waiting for about 10 seconds before giving up...what you would be increasing to 30 seconds...just checking, this seems to be a lot of time to me :thinking:

Reply:
Interestingly doing the math by hand I also expected to get a wait time of 10 seconds overall from code inspection and doubling every time starting with 10 milliseconds, but the canton logs showed the 10 retries occurring in just over 5 seconds. This is why I ended up with tripling instead of only doubling.